traffic_editor/gui/building_level.cpp

```diff
@@ -299,7 +328,51 @@ bool BuildingLevel::delete_selected()
     std::remove_if(
       edges.begin(),
       edges.end(),
-      [](const Edge& edge) { return edge.selected; }),
+      [&](const Edge& edge)
+      {
+        if (edge.selected)
```

Comment:
highly recommend doing an early return, ie. 
```
if (!edge.selected) 
  return false;
...
```
It keeps the code flushed to the left, otherwise you'll be reading code shaped like a staircase.